sunpy/instr/fermi.py

```diff
@@ -451,6 +451,11 @@ def met_to_utc(timeinsec):
     """
     Converts Fermi Mission Elapsed Time (MET) in seconds to a datetime object.
 
+    .. warning::
+        This function does not account for leap seconds and can not be fixed in
+        SunPy's 0.9.x series of releases. However, it is fixed in the next major version
+        coming by May 2019.
```

Comment:
I would say this will be fixed in 1.0, putting release dates in the docs makes me nervous :p

Reply:
Sorted mate